### ISSUE TITLE
[Feature request] Feature Importance for Multi-Class classification

### ISSUE THREAD

**Issue by ChrisZheng1985** [dmlc/xgboost#6241]
Hi,

First of all, really appreciate the xgboost developer here as you guys are awesome and xgboost is a great tools for us either for educational purpose or solving real world problem.

As a lot of you or the users could have this already, but basically when we do multi-class classification, we would also like to see the feature importance for each predicted classes, instead of just the 'average out' feature importance.

So to formalize the request here, if would be great that, when we conduct classification using either 'multi:softmax' or 'multi:softprob', we would like to have:

1. the feature importance as a array of array
2. each predicted class has its own feature importance

best,
Chris
